src/CoreWCF.Metadata/tests/CoreWCF.Metadata.Tests.csproj

```diff
@@ -0,0 +1,59 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Tests only. I did originally plan on creating a package with this name which depended on Primitives, but I ended up with Primitives needing to depend on Metadata types which meant it would always get pulled in anyway.  I tried moving just the needed types into Primitives but that ended up pulling across 90% of the code, so I folded the implementation into Primitives. I kept the tests as a separate project as it depends on Http, NetTcp, and anything else which we build which provides WSDL support.